pkg/prowgen/prowgen.go

```diff
@@ -243,7 +242,7 @@ func generatePresubmitForTest(name string, info *ProwgenInfo, podSpec *corev1.Po
 	return &prowconfig.Presubmit{
 		JobBase:   base,
 		AlwaysRun: runIfChanged == "" && skipIfOnlyChanged == "",
-		Brancher:  prowconfig.Brancher{Branches: sets.NewString(ExactlyBranch(info.Branch), FeatureBranch(info.Branch)).List()},
+		Brancher:  prowconfig.Brancher{Branches: sets.NewString(jc.ExactlyBranch(info.Branch), jc.FeatureBranch(info.Branch)).List()},
```

Comment:
This is the part that I was not sure: In some cases, we need more than `ExactlyBranch`.